### PR TITLE
Explain use with django-rewrite-external-links

### PR DESCRIPTION
**Pull request by henrahmagix**
Done here instead of https://github.com/incuna/django-rewrite-external-links/pull/10

Fix https://github.com/incuna/django-rewrite-external-links/issues/9

@KevinEtienne Please merge, ta!

### CODE REVIEW

**Comment by KevinEtienne** [comment]
:+1:

File: README.rst [hunk 1]
```diff
@@ -39,6 +39,14 @@ Add the tables to the db::
 
 **Warning**: An initial data fixture is included that creates an admin_sso.Assignment to assign any user with an incuna.com email to the Admin user.
 
+If django-rewrite-external-links_ is being used, make sure to allow anonymous access to the external link redirect page by adding the following in ``settings.py``::
```
Comment (KevinEtienne) [inline README.rst]: I think I would make it more generic and say: to allow anonymous access links as for example external links you can use `LOGIN_EXEMPT_URLS`.

Comment (KevinEtienne) [inline README.rst]: Like this we're not enforcing to use any other libraries

Comment (henrahmagix) [inline README.rst]: @KevinEtienne Done in 5e9acd0 =)

File: README.rst [hunk 2]
```diff
@@ -39,6 +39,12 @@ Add the tables to the db::
 
 **Warning**: An initial data fixture is included that creates an admin_sso.Assignment to assign any user with an incuna.com email to the Admin user.
 
+To allow anonymous access to links, like external link rewrites, add ``LOGIN_EXEMPT_URLS`` in ``settings.py``::
```
Comment (meshy) [inline README.rst]: to ~~links~~ **urls**

File: README.rst [hunk 3]
```diff
@@ -39,6 +39,12 @@ Add the tables to the db::
...
+
+    LOGIN_EXEMPT_URLS = [
+        r'^external-link/',
```
Comment (meshy) [inline README.rst]: `r'^about/', ` might be a more generic example.

File: README.rst (hunk 2 above)
Comment (henrahmagix) [inline README.rst]: Done in 54261bb7949d236e381fdd14682a3d2302d78aa5

File: README.rst (hunk 3 above)
Comment (henrahmagix) [inline README.rst]: Done in 8f67559e172a8248f1fac02fcb54277a6b0ac877

File: README.rst [hunk 4]
```diff
@@ -39,6 +39,12 @@ Add the tables to the db::
 
 **Warning**: An initial data fixture is included that creates an admin_sso.Assignment to assign any user with an incuna.com email to the Admin user.
 
+To allow anonymous access to urls, like external link rewrites, add ``LOGIN_EXEMPT_URLS`` in ``settings.py``::
```
Comment (meshy) [inline README.rst]: "external link rewrites" now seems out of place

Comment (henrahmagix) [inline README.rst]: I'd like to mention a common thing that you would want to allow anonymous access to, which is inaccessible by default when using `LoginRequiredMiddleware `.

Comment (meshy) [inline README.rst]: like `/about/`?

Comment (meshy) [inline README.rst]: Actually, just say that: "To allow anonymous access to urls inaccessible by default when using `LoginRequiredMiddleware`.

Comment (henrahmagix) [inline README.rst]: Ah, haha, of course!

**Comment by henrahmagix** [comment]
@KevinEtienne @meshy Do you mind if I squash these commits down to one? The history isn't very meaningful =)

File: README.rst (hunk 4 above)
Comment (henrahmagix) [inline README.rst]: Done in 10d594a

**Comment by meshy** [comment]
Not my call `;)`

**Comment by KevinEtienne** [comment]
Go ahead :)

**Comment by henrahmagix** [comment]
Rebased =)

**Comment by KevinEtienne** [comment]
Thank you @henrahmagix and @meshy :)

**Comment by meshy** [comment]
:+1: :cake:

**Comment by henrahmagix** [comment]
Thanks!

**Pull request merged**